README.md

```diff
@@ -19,11 +19,11 @@ Yep! Macmoji substituions are ordinary text substituions. Double click an entry
 
 Some fun recommendations:
 
-| replace | with |
-| :------- | :------- |
-| ehh | ¯\_(ツ)_/¯ |
-| tflip | (╯°□°）╯︵ ┻━┻ |
-| eml | your@email.com |
+| replace  | with           |
+| :------- | :------------- |
+| ehh      | ¯\\\_(ツ)_/¯   |
```

Comment:
Oh I see, the first one is being escaped. Got it! @mitchellhenke mind adding that second backslash and I'll merge :)